defaults.go

```diff
@@ -82,7 +82,7 @@ func (dns *Msg) SetIxfr(z string, serial uint32, ns, mbox string) *Msg {
 	dns.Question = make([]Question, 1)
 	dns.Ns = make([]RR, 1)
 	s := new(SOA)
-	s.Hdr = RR_Header{z, TypeSOA, ClassINET, defaultTtl, 0}
+	s.Hdr = RR_Header{z, TypeSOA, ClassINET, defaultTTL, 0}
```

Comment:
The Ttl -> TTL ship has sailed. Go DNS uses Ttl (although TTL would have been better in hindsight)